libs/angular-remote-components/src/lib/components/slot/slot.component.ts

```diff
@@ -24,6 +25,10 @@ import { RemoteComponentConfig } from '../../model/remote-component-config.model
 export class SlotComponent implements OnInit, OnDestroy {
   @Input()
   name!: string
+  @Input()
+  inputs: Record<string, unknown> = {}
+  @Input()
+  outputs: Record<string, any> = {}
 
```

Comment:
You should also make sure that changes are propagated to RemoteComponents and that the change detection of the remote component is triggered when an input is changing. For the non-webcomponent case you should consider using https://v17.angular.io/api/core/ComponentRef#setinput to set the values in the component because it ensures change detection